test/network/test_conv3d.py

```diff
@@ -0,0 +1,184 @@
+### Toy example to test Conv3dConnection (the dataset used is MNIST but with a dimension replicated 
+### for each image (each sample has size (28, 28, 28))
```

Comment:
This code should be at MNIST examples. 
Test directory contain tests scripts that perform a integrity check for the models.